horizon/scripts/onms-elastic-init.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+#
+# Intended to be used as part of an InitContainer expecting to run to configure Elastic settings
+#
+# External environment variables used by this script:
```

Comment:
Can you add a list of variables used in this script.

FYI, there is a limit on config map size (by default it is 1mb).